lib/filter/tests/Makefile.am

```diff
@@ -29,6 +30,15 @@ lib_filter_tests_test_filters_facility_SOURCES = 			\
 	lib/filter/tests/test_filters_common.c \
 	lib/filter/tests/test_filters_common.h
 
+lib_filter_tests_test_filters_level_new_CFLAGS     = $(TEST_CFLAGS) \
+	-I${top_srcdir}/lib/filter/tests
+lib_filter_tests_test_filters_level_new_LDADD      = $(TEST_LDADD)  \
+	$(PREOPEN_SYSLOGFORMAT)
+lib_filter_tests_test_filters_level_new_SOURCES = 			\
+	lib/filter/tests/test_filters_level_new.c \
```

Comment:
What does the `_new` express in the name?

Reply:
the _new comes from filter name filter_level_new, so it's not my name :-)